pallets/common/src/lib.rs

```diff
@@ -10,3 +10,42 @@ pub use context::Context;
 
 pub mod batch_dispatch_info;
 pub use batch_dispatch_info::BatchDispatchInfo;
+
+use core::convert::From;
+use polymesh_primitives::IdentityId;
+
+/// It defines the valid issuers for Systematic Claims.
+///
+/// Systematic claims are claims generated by the system itself, and they are removed automatically
+/// too.
+///
+/// There is just one Systematic claim at the moment, and It is generated by two issuers.
+/// It is a CDD claim that is assigned to members of specific groups:
+/// * Governance Committee: Each member of the committee has a CDD claim generated by
+/// `SystematicIssuers::Committee` in order to allow it to operate independently of
+/// claims generated by CDD trusted providers. Using that claim, GC members could operate even if
+/// their CDD claim's issuer has been revoked.
+/// * CDD Service Providers: Every CDD providers has a CDD claim generated by
+/// `SystematicIssuers::CDDProvider` group, in order to avoid self-generated claim issue.
```

Comment:
Out of interest, is there a technical reason to have a different systematic issuer for CDDs vs. GC members? In my head they are both being issued by "Polymesh" or arguably the governance committee in a systematic sense.

Reply:
I guess it means that if you have a CDD provider that is also a GC member, then they would have two claims (one from Committee, one from CDDProvider). If they left one of the two groups, there other claim would remain so I guess it makes that logic simpler which is good.

Reply:
Yes, that is. Actually I added a test case where an Identity is in CDD & GC group at the same time.
Test: `gc_and_cdd_with_systematic_cdd_claims` in `identity_test.rs`